pkg/apis/kops/v1alpha2/networking.go

```diff
@@ -108,8 +108,8 @@ type CalicoNetworkingSpec struct {
 	// AllowIPForwarding enable ip_forwarding setting within the container namespace.
 	// (default: false)
 	AllowIPForwarding bool `json:"allowIPForwarding,omitempty"`
-	// AWSSrcDstCheck enables/disables ENI source/destination checks (AWS only)
-	// Options: Disable (default), Enable, or DoNothing
+	// AWSSrcDstCheck enables/disables ENI source/destination checks (AWS IPv4 only)
+	// Options: Disable (default for IPv4), Enable, or DoNothing
```

Comment:
Isn't the above `(AWS IPv4 only)` enough?
```suggestion
	// Options: Disable (default), Enable, or DoNothing
```

Reply:
That could imply one has to override the default when on IPv6

Reply:
To me the initial versions implies that there is some default for IPv4 and undefined for IPv6 and has to be set by the operator.
Would be nice to also add validation to not be able to set it for IPv6 at all.

Reply:
This PR includes validation preventing it from being set to anything other than `DoNothing` on IPv6.

Reply:
K, I'm good/